Vector/Views/RoomBubbleList/RoomOutgoingAttachmentWithoutSenderInfoBubbleCell.h

```diff
@@ -15,6 +15,7 @@
  */
 
 #import "MXKRoomOutgoingAttachmentWithoutSenderInfoBubbleCell.h"
+#import "RoomOutgoingAttachmentBubbleCell.h"
```

Comment:
Like this, this is the normal scheme when a class inherits from `RoomOutgoingAttachmentBubbleCell`.
Even if it is not the case in the obj-c implementation, this is what we want functionally.